pyr2.html

```diff
@@ -58,31 +56,38 @@
 var a13 = qnbank[Qline][7];
 var snod = qnbank[Qline][8];
 var a14 = qnbank[Qline][9];
-document.write("<DL>");
+
+document.innerHTML="<DL>";
```

Comment:
Additionally `DL` does not seem like an ideal element for this content. How about `ARTICLE` or `SECTION`?